docs/account.mdx

```diff
@@ -0,0 +1,630 @@
+---
+title: Account operations
+description: Using CLI to manage your tokens
+---
+
+import DocCard from '@theme/DocCard';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+
+# `account`
+
+`account` command is used for consensus or ParaTime-layer on-chain actions
+associated with accounts:
+
+- getting the account balance including delegations,
+- transferring tokens between the accounts,
+- delegating or undelegating tokens to or from validators (*staking*),
+- depositing and withdrawing tokens to or from a ParaTime,
+- changing permission policy of the account,
```

Comment:
What is this about, what kind of "permission policy"?

Reply:
allowance transaction